lib/puppet/tab.ts

```diff
@@ -23,6 +25,18 @@ export default class Tab implements TabActor {
     this.frames = frames;
   }
 
+  async _initPlaywrightUtils() {
+    const script = `(() => {
+      ${getStyleElementUtil.toString()}
+      ${hideElementsUtil.toString()}
+      window.autoconsentTestUtils = {
+        getStyleElementUtil,
+        hideElementsUtil,
+      };
+    })();`;
+    return this.frames[0].evaluate(script);
```

Comment:
Can we not inject these functions on demand when we do the actual hiding? This prevents us from having to do injection when the page loads, and also means we don't have to attach things to global scope.

Note this also might break if the page navigates after we attached to it.